hack/deploy_postgres.sh

```diff
@@ -10,7 +10,7 @@ kubectl wait --timeout=60s --for=condition=ready pod -n postgres -l app=postgres
 # Wait for extra seconds
 sleep 5
 
-dbs=("cluster_manager" "model_manager" "file_manager" "job_manager" "user_manager" "dex")
+dbs=("cluster_manager" "model_manager" "file_manager" "job_manager" "user_manager" "dex" "vector_store_manager")
```

Comment:
Could you also update https://github.com/llm-operator/llm-operator/blob/main/aws/scripts/setup- databases.sh#L10 ?